mandatory/1-fix-functions.js

```diff
@@ -10,8 +10,8 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
+function getMood(bool) {
+  let isHappy = bool;
```

Comment:
Hi Appolin, I hope you don't mind, I'm doing a quick review of your code because I couldn't see any submissions from my Group D yet and I'm trying to complete the assignments... 
Good job, this code will definitely work. To use less lines of code, you could have placed isHappy in the function parameter directly so that you didn't need to declare the extra bool variable. 
I like that you did the "if" test with just the variable name, rather than writing it out fully if (isHappy === true) - your way is more efficient.